### DISCUSSION TITLE
More examples and use cases needed

### DISCUSSION THREAD

**Discussion by roofiq** [devlooped/nugetizer#46] [Q&A] [answered]
Hey,

As a not advanced person when it comes to compiling  .Net apps i would really appreciate if we could have more examples and scenarios added.
Description is prettty solid but as this tool is not heavily use its hard to find some error that popped up or more. I liked how you approached it on stackoverflow post - i will include link later.

One of examples is this error. Trying to test packing and this popps out. Screen attached.
I believe im missing some properties in csproj?

![IMG_20201209_005312.jpg](https://user-images.githubusercontent.com/44588684/101555736-82bba280-39b9-11eb-8286-7651493e8128.jpg)

Thanks,
Rafał

**Comment by kzu** [comment, marked as answer]
The "Project is not packable" means you don't have a `PackageId` specified (or `IsPackable=true`) in the project file.

I'll add a link and start monitoring SO questions on #nugetizer so I can properly answer more questions there too. I also enabled discussions on this project, so I'll move this over there.
